be/test/cache/block_cache_test.cpp

```diff
@@ -0,0 +1,77 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
shall add CacheInputStream test case to ensure the stream behavior is as expected. e.g. seek to random place and still can read arbitrary length of data correctly, which means give any (offset, length) pair, the CacheInputStream is well behaviored.

Reply:
> shall add CacheInputStream test case to ensure the stream behavior is as expected. e.g. seek to random place and still can read arbitrary length of data correctly, which means give any (offset, length) pair, the CacheInputStream is functioning well.

OK, I will add some ut in this PR.